app/src/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2245,6 +2244,14 @@ private int getCurrentFormID() {
         return mFormController.getFormID();
     }
 
+    public void handleFormSaveError(Exception e) {
+        if (e instanceof IllegalStateException) {
+            // TODO PLM: send this error to HQ as a app build error, most likely a user level issue.
```

Comment:
Note the TODO. Don't really want to do this right now because it is extra work and this isn't even setup on HQ yet.